infra/core/ai/cognitiveservices.bicep

```diff
@@ -6,11 +6,21 @@ param tags object = {}
 param customSubDomainName string = name
 param deployments array = []
 param kind string = 'OpenAI'
+
+@allowed(['Enabled', 'Disabled'])
 param publicNetworkAccess string = 'Enabled'
 param sku object = {
   name: 'S0'
 }
 
+param allowHosts array = []
```

Comment:
I prefer allowedHosts since allowHosts sounds like a boolean. And per earlier comment, I prefer allowedIPAddresses